classify_images.py

```diff
@@ -0,0 +1,68 @@
+import os
+import shutil
+import geopandas as gpd
+
+
+def get_image_name(gdf: gpd.GeoDataFrame) -> list:
+    gdf.index = [i for i in range(len(gdf))]
+    # Names
+    name_list = gdf["NOM_IMAGE_"].values
+    names_compararison = []
+    for name in name_list:
+        name_split = name.split("_")
+        name_split.pop()
+        name_numbered = "_".join(name_split)
+        names_compararison.append(name_numbered)
+    # name_list = []
+    return names_compararison
+
+
+def main():
+
+    raster_folder = r"/folder/where/the/photos/are/saved"
```

Comment:
You can put them in Global variables in upper case.

Reply:
Sure!